### ISSUE TITLE
Remove footers and/or headers for particular pages

### ISSUE THREAD

**Issue by fw-aaron** [trade-informatics/caracal#139]
I set `docx.page_numbers true`, but on some pages I would like not to have page numbers (and the same with the header). How do I do this with caracal?

**Comment by WWGate** [comment]
@fw-aaron did you find a solution?

**Comment by fw-aaron** [comment]
@WWGate No, sir. I told my client that in this day of technology, we aren't quite advanced enough to do that, and I told him where he could put his page numbers.

**Comment by WWGate** [comment]
thank you for the reply @fw-aaron 
I used Sablon gem, It seems promising!
